multimedia/M2VDownsizer/Files/M2VDownsizer.1

```diff
@@ -0,0 +1,73 @@
+.SH NAME
+\fBM2VDownsizer\fP - Re-compresses MPEG-2 video streams
+.SH SYNOPSIS
+\fBM2VDownsizer\fP <lowering factor> <quality 0=fast...3=high quality>
+<input m2v size in bytes>
+.SH DESCRIPTION
+\fBM2VDownsizer\fP is a tool that re-compresses MPEG-2 (or m2v) video streams.
+In other words, \fBM2VDownsizer\fP will perform a complete decompression and
+recompression of MPEG-2 video streams for the best possible video quality.
+.PP
+\fBM2VDownsizer\fP accepts the raw MPEG-2 (or m2v) video data (not VOB) from
+the standard input and writes the recompressed video frames to the standard
+output.
+.PP
+Be forewarned! Decompression and recompression  could take hours for large
+MPEG-2 video streams!  It is recommended that one first tries transcoding
+MPEG-2 streams using \fBM2VRequantiser\fP since transcoding is a lot faster and
+produces excellent results for most MPEG-2 video streams. \fBM2VDownsizer\fP is
+only intended to be used for large MPEG-2 video streams when transcoding does
+not produce acceptable results.
+.SH OPTIONS
+\fBM2VDownsizer\fP takes three arguments and does not have any options. The
+usage is:
+.PP
+.nf
+.fam C
+    $ M2VDownsizer <lowering factor> <quality 0=fast...3=high quality>
+    <input m2v size in bytes>
+.fam T
+.fi
+.PP
+The first argument is a floating point value specifying the lowering factor for
+re-compression. The higher the lowering factor, the higher the re-compression
+will be. The second argument is an integer value from 0 to 3 specifying the
+quality of the re-compressed MPEG-2 output video (0=lowest quality and
+3=highest quality). The third argument is the size of the MPEG-2 input video in
+bytes. (Since the MPEG-2 video is streamed into \fBM2VDownsizer\fP, the program
+is unable to determine the MPEG-2 input video size).
+.PP
+To shrink MPEG-2 video from a DVD-9 disc to fit on a DVD-5 disc, a lowering
+factor of 1.5 works well in most cases. Alternatively, use the following
+formula to more precisely calculate the lowering factor:
+.PP
+.nf
+.fam C
+    lowering factor = (( Total m2v video size in bytes / 
+    (4700000000 - Total audio size in bytes)) * 1.04
+.fam T
+.fi
+.SH EXAMPLE
+The following command would re-compress 'original.m2v', whose size is
+5,974,039,472 bytes, by a lowering factor of 1.46 with the highest quality of
+3:
+.PP
+.nf
+.fam C
+    $ M2VDownsizer 1.46 3 5974039472 < original.m2v > re-compressed.m2v
+.fam T
+.fi
+.PP
+It will take a long time for \fBM2VDownsizer\fP to completely decompress and
+recompress the MPEG-2 video in the above example.  Time to take a long coffee
+break!
+.SH SEE ALSO
+\fBM2VRequantiser\fP(1)
+.SH AUTHOR
+The author is unkown.  \fBM2VDownsizer\fP was released as open source software
+on the website for Metakine Inc., the developer of DVDRemaster.
+.PP
+This manual page was written by Robert Kennedy <amtor@hotmail.com> for the
+Macports project (but may be used by others).
```

Comment:
```suggestion
MacPorts project (but may be used by others).
```

Reply:
P.S.  I have also updated the man page for the sister port - `M2VRequantiser`.  I will submit it via a PR next week and will make the same change.